README.md

```diff
@@ -680,6 +692,7 @@ hop = false
 ```lua
 indent_blankline = {
     enabled = true,
+	scope_color = "", -- catppuccin color (eg. lavendar) Default: text
```

Comment:
Wrong indentation, what did you use to format readme.md?

Reply:
Prettier. I turned my formatter off the the most recent commit.